contrib/nydus-overlayfs/cmd/main.go

```diff
@@ -13,7 +13,8 @@ import (
 )
 
 const (
-	extraOptionKey = "extraoption="
+	extraOptionKey      = "extraoption="
+	kataVolumeOptionKey = "io.katacontainers.volume="
```

Comment:
Can add some comments to explain why we ignore the option key here?

Reply:
Yeah, I'm confused too.

Reply:
This is to make use of the `KataVirtualVolume` abstraction from
https://github.com/kata-containers/kata-containers/blob/main/src/libs/kata-types/src/mount.rs#L233